commit 7f2cea360936f8276e0c241e5d84099311b7e90f
Author: Jay Logue <jaylogue@google.com>
Date:   Fri Aug 30 07:49:14 2019 -0700

    openweave-esp32-lwip-patch-0009 : Enable custom pbufs
    
    Enabled support for LwIP's custom pbufs, which are required for certain Weave
    functionality.

diff --git a/port/esp32/include/lwipopts.h b/port/esp32/include/lwipopts.h
index 8b5f2ad..0bbc4fb 100644
--- a/port/esp32/include/lwipopts.h
+++ b/port/esp32/include/lwipopts.h
@@ -396,6 +396,12 @@
    ----------------------------------
 */
 
+/**
+ * Enable support for custom PBUFs.
+ */
+#define LWIP_SUPPORT_CUSTOM_PBUF 1
+
+
 /*
    ------------------------------------------------
    ---------- Network Interfaces options ----------
